packages/database/src/migrations/20200921010041-FixLaosSchoolsSchoolIndicatorsEiESubNationalLevelsMapOverlaysEntityAggregation-modifies-data.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const NUMERATOR_PATH = [
+  'measureBuilderConfig',
+  'measureBuilders',
+  'numerator',
+  'measureBuilderConfig',
+  'entityAggregation',
+];
+const DENOMINATOR_PATH = [
+  'measureBuilderConfig',
+  'measureBuilders',
+  'denominator',
+  'measureBuilderConfig',
+  'entityAggregation',
+];
+
+const NEW_AGGREGATION_TYPE = 'COUNT_PER_ORG_GROUP'; //Used to be SUM_PER_ORG_GROUP
+
+const NEW_NUMERATOR_CONFIG = {
+  condition: {
+    operator: '=',
+    value: 'Yes',
+  },
+};
+const NEW_DENOMINATOR_CONFIG = {
+  condition: '*',
+};
+
+exports.up = function(db) {
```

Comment:
The current config in dev is:
https://github.com/beyondessential/tupaia-backlog/issues/1312#issuecomment-695875916

The config for for numerator was updated recently to fit the new code in `aggregateAnalytics.js`, but seems like the config for the denominator was missed. Also, there were some new overlays added to the sub national groups and was using the old config (basically a migration config). So this migration is for updating both the numerator/denominator config to match the new code in `aggregateAnalytics.js`